### PR TITLE
[do not merge] JBPM-6659 - Update details pages to take full screen

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java [hunk 1]
```diff
@@ -71,17 +70,20 @@ public void setServerTemplateSelectorMenuBuilder(final ServerTemplateSelectorMen
...
-        setSelectedServerTemplate(serverTemplateSelected.getServerTemplateId());
+        setSelectedServerTemplate(serverTemplateSelected.getServerTemplateId(), false);
     }
 
     public String getSelectedServerTemplate() {
         return selectedServerTemplate;
     }
 
-    protected void setSelectedServerTemplate(final String selectedServerTemplate) {
+    protected void setSelectedServerTemplate(final String selectedServerTemplate, final Boolean forceRefresh) {
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java]: Demo seemed successful :)
One comment I have at this point is that this two argument setter looks more as a workaround then the way to go when we start redesigning, but I have no suggestions at the moment.

File: jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenterTest.java [hunk 2]
```diff
@@ -302,7 +303,7 @@ public void testJobSelectionWithDetailsClosed() {
         presenter.selectJob(job,
                             closed);
 
-        verify(placeManager).goTo(JobDetailsPresenter.SCREEN_ID);
```
Comment (nmirasch) [inline jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenterTest.java]: @cristianonicolai could be the JobDetailsPresenter import removed?

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java [hunk 3]
```diff
@@ -586,7 +587,7 @@ public void testExitedTaskSelection() {
         getPresenter().selectTask(taskSummary,
                                   closed);
 
-        verify(placeManager).goTo(TaskDetailsMultiPresenter.SCREEN_ID);
```
Comment (nmirasch) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java]: @cristianonicolai same with TaskDetailsMultiPresenter unused import

**Comment by nmirasch** [comment]
@cristianonicolai Is there any reason to not to include AdvancedProcessDefDetailsMultiPresenter.SCREEN_ID at PerspectiveIds ?

@cristianonicolai Good work about perspective Ids centralization!

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractScreenListPresenter.java]: I agree it isn't the nicest change but... as you said, there will be probably more changes related to this once we actually get the new designs in.

**Comment by cristianonicolai** [comment]
@nmirasch thanks, fixed all :)

**Pull request closed without merging**